src/Sylius/Bundle/UserBundle/Resources/config/services.xml

```diff
@@ -179,12 +182,12 @@
         <service id="sylius.form.type.user_change_password" class="%sylius.form.type.user_change_password.class%">
             <tag name="form.type" alias="sylius_user_change_password" />
         </service>
-        <service id="sylius.form.type.gender" class="Sylius\Bundle\UserBundle\Form\Type\GenderType">
+        <service id="sylius.form.type.gender" class="%sylius.form.type.gender.class%">
```

Comment:
You can always open a RFC about this. But for now, we have convention to use class names as parameters and I found it more appropriate to stay consistent then to change practise without discussion, just to follow some changes.